controllers/communityController.js

```diff
@@ -27,28 +27,31 @@ module.exports.getCommunity = async function (req, res) {
 }
 
 module.exports.getMember = async function(req, res) {
-  const id = req.params.id;
+  const requestedUserId = req.params.id;
 
   try {
-    const isCurrentUser = req.user.id === id;
+    const isCurrentUser = req.user.id === requestedUserId;
 
     if (isCurrentUser) {
       res.redirect(`/account`);
       return;
     }
 
-    const user = await utils.userInfos(id, req.user.id === id);
-
-    if (!user.userInfos) {
-      req.flash('error', `Inexistante (nécessaire pour créer le compte mail) Vous pouvez créer la fiche sur Github`);
-      res.redirect(`/community`);
-      return;
+    const user = await utils.userInfos(requestedUserId, isCurrentUser);
+    
+    const hasGithubFile = user.userInfos;
+    const hasEmailAddress = (user.emailInfos || user.redirections.length > 0);
+    if (!hasGithubFile && !hasEmailAddress) {	
+      req.flash('error', `Il n'y a pas d'utilisateurs avec ce compte mail. Vous pouvez commencez par créer une fiche sur Github pour la personne <a href="/onboarding">en cliquant ici</a>.`);	
+      res.redirect(`/community`);	
+      return;	
     }
     const marrainageStateResponse = await knex('marrainage').select()
         .where({ username: id });
```

Comment:
```suggestion
        .where({ username: requestedUserId });
```
Souci de merge j'imagine